erc1155/src/lib.rs

```diff
@@ -61,31 +63,23 @@ impl ERC1155Token {
 }
 
 impl ERC1155TokenBase for ERC1155Token {
-    fn balance_of(&self, account: &ActorId, id: &u128) -> u128 {
+    fn balance_of(&self, account: &ActorId, id: &TokenId) -> u128 {
         self.get_balance(account, id)
```

Comment:
It seems that defining `balance_of` and `get_balance` at the same time is redundant, because the `balance_of` just calls the `get_balance`